parm/config/gefs/config.resources

```diff
@@ -224,11 +218,39 @@ case ${step} in
     export is_exclusive=True
     ;;
 
-  "stage_ic")
-    export wtime_stage_ic="00:15:00"
-    export npe_stage_ic=1
-    export npe_node_stage_ic=1
-    export nth_stage_ic=1
+  "wavepostsbs")
+    export wtime_wavepostsbs="03:00:00"
+    export npe_wavepostsbs=8
```

Comment:
At some point updating this number to be optimal for GEFS should be considered - not necessarily now but just wanted to point it out to @sbanihash and others.  Does the wave post sbs already take care of ensemble members for GEFS?  If so, this could be a larger number depending on how fast you want it to process the full output.

Reply:
Each member has its own job, so there is no impact io wall-clock. And we want to refactor it to be more like our other post/product jobs and only process one forecast hour (see #2290).

Reply:
If each member has it's own job, I think this could be 1 instead of 8.

Reply:
Will test that, but 8 is used for GFS, which is where I copied this block from.

Reply:
Seems to work, but I only tested C48 so I'm still a bit leery high resolution would have an issue. Will got with whatever you recommend (and why would it be different than GFS?)

Reply:
@WalterKolczynski-NOAA This is a MPMD program. For GFSv16 we had multiple output grids and therefore had 8 processors.  For GEFSv13, you only have 1 output grid.   This might need to be accompanied by an increase in memory for the overall job for say the mx025 but  unless things have changed, or you were processing for every ensemble member (in which case I'd have recommended more processors), I think 1 for GEFS is the correct decision.

Reply:
Okay, will do.

Reply:
Changed